docs/core/tutorials/top-level-templates.md

```diff
@@ -96,92 +96,52 @@ You can also add a [`<Using>`](../project-sdk/msbuild-props.md#using) item with
+### Use the old program style (do not use top level statements) in Visual Studio
 
-When you create a new console project in Visual Studio, you're prompted with a dropdown box that identifies which target framework you want to use. Change that value to **5.0**. After the project is created, edit the project file to change it back to **6.0**.
+01. When you create a new project, the setup steps will navigate to the **Additional information** setup page. On this page, select "Do not use top-level statements" check box.
 
-01. When you create a new project, the setup steps will navigate to the **Additional information** setup page. On this page, change the framework setting from **.NET 6.0 (Long-term support)** to **.NET 5.0**, and then select the **Create** button.
+    :::image type="content" source="media/top-level-templates/vs-additional-information.png" alt-text="Visual Studio "Do not use top-level statements" check box":::
 
-    :::image type="content" source="media/top-level-templates/vs-additional-information.png" alt-text="Visual Studio select the target .NET Framework 5.0":::
+01. After your project is created, the `Program.cs` content is as following:
 
-01. After your project is created, find the **Project Explorer** pane. Double-click on the project file and change `<TargetFramework>net5.0</TargetFramework>` to `<TargetFramework>net6.0</TargetFramework>`.
-
-    Here's a file diff that illustrates the changes:
-
-    ```diff
-    <Project Sdk="Microsoft.NET.Sdk">
-
-      <PropertyGroup>
-        <OutputType>Exe</OutputType>
-    -   <TargetFramework>net5.0</TargetFramework>
-    +   <TargetFramework>net6.0</TargetFramework>
-      </PropertyGroup>
-
-    </Project>
+    ```csharp
+    namespace MyProject;
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, World!");
+        }
+    }
     ```
 
-    Alternatively, you can right-click on the project in the **Solution Explorer** pane, and select **Properties**. This opens up a settings page where you can change the **Target framework**.
-
-    :::image type="content" source="media/top-level-templates/vs-project-properties.png" alt-text="Edit the project properties for Visual Studio and set the .NET Framework version.":::
-
-01. Optional step: you can still use some of the newer .NET 6 and C# features by adding the properties for [implicit `using` directives](#implicit-using-directives) and [nullable context](../../csharp/language-reference/compiler-options/language.md#nullable) to the project file.
+> [!NOTE]
+> Visual Studio preserves the value for the options next time you create the project based on the same template, so by default when creating Console App project next time the "Do not use top-level statements" check box will be checked.
+> The content of `Program.cs` file might be different to match code style defined in the global Visual Studio text editor settings or `EditorConfig` file.
```

Comment:
@tdykstra the following links will be helpful here:
- https://learn.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2022
- https://learn.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-csharp-advanced?view=vs-2022

is there a way to add them?